### PR TITLE
two pages pdf

### PR DESCRIPTION
**Pull request by cristinalare**
- added the pdf version displayed on two-pages
- another (maybe temporary) flip book - tbd

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -57,15 +68,20 @@
...
 		</div>
 
 		<div class="hero-container">
 			<div class="hero-content">
 				<h1>Step into the<br/>Pluriverse</h1>
 				<p>
 					Read the guide &nbsp;
-					<a href="/pdf/guide.pdf">[PDF] &nbsp;</a>
-					<a href="https://online.fliphtml5.com/wryqh/bdyf/"> [Flipbook]</a>
+					<a href="/pdf/pluriverse-guide.pdf">[PDF] &nbsp;</a>
```
Comment (owocki) [inline index.html:83]: @cristinalare heads up the site 404s now when u click on the pdf option

Comment (owocki) [reply index.html:83]: https://pluriverse.wtf/

Comment (owocki) [reply index.html:83]: do u have a copy of this new 2 page pdf?  if so i can upload it ot the server at this url

Comment (cristinalare) [reply index.html:83]: yes, right, /pdf is in gitignore. im sending you the pdf rn

**Pull request merged**